pkg/apiserver/registry/stats/nodelatencystats/rest_test.go

```diff
@@ -29,31 +30,61 @@ import (
 )
 
 func TestREST(t *testing.T) {
-	r := NewREST()
+	fakeClock := clocktesting.NewFakeClock(time.Now())
+	r := newRESTWithClock(fakeClock)
 	assert.Equal(t, &statsv1alpha1.NodeLatencyStats{}, r.New())
 	assert.Equal(t, &statsv1alpha1.NodeLatencyStats{}, r.NewList())
 	assert.False(t, r.NamespaceScoped())
 }
 
 func TestRESTCreate(t *testing.T) {
-	summary := &statsv1alpha1.NodeLatencyStats{
-		ObjectMeta:           metav1.ObjectMeta{Name: "node1"},
-		PeerNodeLatencyStats: nil,
-	}
-	expectedObj := &statsv1alpha1.NodeLatencyStats{
-		ObjectMeta:           metav1.ObjectMeta{Name: "node1"},
-		PeerNodeLatencyStats: nil,
-	}
-
-	r := NewREST()
 	ctx := context.Background()
+	now := time.Now()
+	const timeStep = 1 * time.Minute
+	tests := []struct {
+		name        string
+		summary     *statsv1alpha1.NodeLatencyStats
+		expectedObj *statsv1alpha1.NodeLatencyStats
+	}{
+		{
+			name: "create with existing timestamp",
+			summary: &statsv1alpha1.NodeLatencyStats{
+				ObjectMeta:           metav1.ObjectMeta{Name: "node1", CreationTimestamp: metav1.Time{Time: now}},
+				PeerNodeLatencyStats: nil,
+			},
+			expectedObj: &statsv1alpha1.NodeLatencyStats{
+				ObjectMeta:           metav1.ObjectMeta{Name: "node1", CreationTimestamp: metav1.Time{Time: now}},
+				PeerNodeLatencyStats: nil,
+			},
+		},
+		{
+			name: "create with no existing timestamp",
+			summary: &statsv1alpha1.NodeLatencyStats{
+				ObjectMeta:           metav1.ObjectMeta{Name: "node1"},
+				PeerNodeLatencyStats: nil,
+			},
+			expectedObj: &statsv1alpha1.NodeLatencyStats{
+				// the test case advances the clock by timeStep
+				ObjectMeta:           metav1.ObjectMeta{Name: "node1", CreationTimestamp: metav1.Time{Time: now.Add(timeStep)}},
+				PeerNodeLatencyStats: nil,
+			},
+		},
+	}
 
-	obj, err := r.Create(ctx, summary, nil, nil)
-	require.NoError(t, err)
-	assert.Equal(t, expectedObj, obj)
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			fakeClock := clocktesting.NewFakeClock(now)
+			r := newRESTWithClock(fakeClock)
+			fakeClock.SetTime(now.Add(timeStep))
```

Comment:
```suggestion
			fakeClock.Step(timeStep)
```

I may have forgotten to suggest it in my previous review